### DISCUSSION TITLE
"Mapping for sensemaking" in deliberately developmental spaces

### DISCUSSION THREAD

**Discussion by msantolini** [life-itself/community#691] [Research]
This discussion follows a post in #345 which is tied to the issue #688, following a conversation with @nathenf and @laurenwigmore. I figured it could have its own thread, as we are ready to prototype things. 

We are envisioning to put QR codes around the Bergerac Hub that link to google forms where one can say what rooms they are in and answer a simple question about their experience. I think it's a great opportunity to prototype some sensemaking, in the vein of what John Olliver showed us in his research talk. We already have a questionnaire (https://forms.gle/D7kpLz8hNVEPdGri8), but I think we could stick to one question only. For kitchen/dinner room we could use the question on food. For other rooms, my gut feeling would be to ask something about the practice and experience a participant has been involved in in a free text form, and then maybe use 2 scales 1-10 that could be related to how alone vs collective the experience is, and how inner (mediation etc) vs outer (e.g. hackathon) it is. 

My sense would be to prototype a Wilber-like graph with a scatter plot of reports that we would like to see how it "moves" along the inner collective as residencies go, along with textual reports to dive into the experience beyond numbers. 

Question for discussion here: **what would be a good, simple prompting question**?

**Comment by asimong** [comment]
Seems like a nice idea to me. A couple of cautions arise in my mind …

1. It assumes everyone has a smartphone to hand and is familiar with using QR codes. Yes, most of the people we know who will be at the Bergerac Hub will, but for sure, not all. What about those people? Would you envisage a clearly signposted and supported alternative for people who do not have smartphones to hand, or are not able or willing to use them in that way?
2. Is there any prompting question that would be sufficiently context-free to allow the results of historical comparison to be useful? I guess that's one take on the question you are actually asking here.

My discomfort arises around the question, why would you think that a QR-and-smartphone approach would be better than a person-to-person interview? Less costly in terms of researcher time, for sure. But what are the other potential biases that might be introduced?

Another related pointed question: how could, or might, sensemaking be affected by the means/medium/mode by which the data are gathered? Do you have any leads on that question? It's the kind of question I would be asking if I were examining a dissertation in this area.

**Reply by msantolini** [reply]
Very fair question and points! First, indeed this doesn't replace, but more "parallels" interview options or surveys sent by email (which have their own selection biases). One thought was to have at some point a "documentation station" (think like an ipad in the entrance) with direct access to questionnaire in order to support those not having an easy entry-point towards the QR-code system. For the other question "how could, or might, sensemaking be affected by the means/medium/mode by which the data are gathered?", I'd be happy to hear more from John on this, as indeed any method (even ethnographic participant observation) introduces some form of bias, and only an ecology of methods can begin to bring some form of wider picture.

**Comment by nathenf** [comment]
We've got to move fast on this @msantolini what do you reckon?

**Reply by baouroux** [reply]
Hi @msantolini @nathenf @laurenwigmore,
I'd love to contribute to this, with the similarities with my previous narratives sensemaking client projects, but still on vacation until the 3rd, and I've just lost my original reply 20min of editing from my phone in Github when switching windows to attach a file!

Lots of great points to follow up from Simon, and provide some quick design suggestions that can be implemented on something like Google Forms.

I'm afraid though I do need to run now for the evening, so I'll try to send more tomorrow morning, otherwise I could even be free for a call tomorrow or Wednesday.

Meantime, this is the platform I used extensively in collaboration with Dave Snowden, with background and design overview: 
https://thecynefin.co/how-to-use-data-collection-analysis-tool/

Even Googling 'Sensemaking design guidelines' provides some useful sources, and checking out Cynthia Kurtz's NarraFirma (free) platform has excellent design resources https://narrafirma.com/
